pkg/utils/utils.go

```diff
@@ -34,3 +34,11 @@ func RemoveString(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+func ClearMap(inputMap map[string]string) {
```

Comment:
This is used only in tests, please move it to tests module or even to the test using it directly as a private function until we have multiple users.

Reply:
done